docs/user/builds.rst

```diff
@@ -68,6 +68,46 @@ The following are the pre-defined jobs executed by Read the Docs:
     it's possible to run user-defined commands and :doc:`customize the build process <build-customization>`.
 
 
+Cancelling builds
```

Comment:
This heading can definitely be improved to be explicit about what this is about. It's definitely [explanation](https://diataxis.fr/explanation/#the-language-of-explanation):

```suggestion
When to cancel builds
```